cpp/include/cudf/unary.hpp

```diff
@@ -31,6 +33,65 @@ namespace cudf {
  * @brief Column APIs for unary ops
  */
 
+/**
+ * @brief Convert a floating-point value to fixed-point
+ *
+ * @tparam Fixed The fixed-point type we are converting to
+ * @tparam Floating The floating-point type we are converting from
+ * @param floating The floating-point value to convert
+ * @param scale The desired scale of the fixed-point value
+ * @return The converted to fixed-point value
```

Comment:
Suggestion. Avoid the royal we, and be consistent with periods.

```suggestion
 * @tparam Fixed The fixed-point type to convert to.
 * @tparam Floating The floating-point type to convert from.
 * @param floating The floating-point value to convert.
 * @param scale The desired scale of the fixed-point value.
 * @return The converted fixed-point value.
```

Reply:
We've been _trying_ to consistently **not** use periods here and only use them with complete sentences like in the description. 
Some of the less changed, older code still has periods for the doxygen tag text.

Reply:
I'll remove the we's